README.md

```diff
@@ -1,4 +1,4 @@
-# README
+# qna [![CircleCI](https://circleci.com/gh/JDTeamAcetabulum/qna.svg?style=shield)](https://circleci.com/gh/JDTeamAcetabulum/qna) [![Coverage Status](https://coveralls.io/repos/github/JDTeamAcetabulum/qna/badge.svg?branch=strburst-circleci-config)](https://coveralls.io/github/JDTeamAcetabulum/qna?branch=strburst-circleci-config)
```

Comment:
You probably want the coveralls link to point to the coverage of the master branch here.